django/test/utils.py

```diff
@@ -850,3 +856,35 @@ def register_lookup(field, *lookups, lookup_name=None):
     finally:
         for lookup in lookups:
             field._unregister_lookup(lookup, lookup_name)
+
+
+class PlainPasswordHasher(BasePasswordHasher):
```

Comment:
Inspired by https://github.com/peopledoc/django-plainpasswordhasher

Which is mentioned in Trac